packages/belos/tpetra/src/BelosTpetraOperator.hpp

```diff
@@ -151,15 +151,15 @@ class TpetraOperator : public Tpetra::Operator<Scalar, LocalOrdinal, GlobalOrdin
       lp_->setInitResVec( vec_X );
     }
     lp_->setProblem( vec_Y, vec_X );
-    Belos::ReturnType ret = solver_->solve();
+    solver_->solve();
```

Comment:
Thanks for the fixes!  Later on, we could think about giving users a way to get the solve status, but that can be for later : - ) .

Reply:
Yes, I thought about it briefly, but no immediate solution presented itself.  I guess we could create another function that could query the result of the last solve... But if the main purpose is to use belos as a preconditioner, the user probably isn't especially concerned about whether it converged.

Reply:
@jennloe It could be useful for inner-outer iterations, to avoid recomputing the residual, but you might need to do that anyway.  We can always not implement optional features ;-) .